src/screens/TopicPageScreen/elements/BottomSheetFollow.tsx

```diff
@@ -89,6 +82,8 @@ const BottomSheetFollow = forwardRef((props: BottomSheetFollowProps, ref: Ref<RB
     setIsFollow(false);
     setMemberCount(memberCount - 1);
     onClose();
+    updateFollowTopicDiscoveryContext(false, {topic_id: topicName}, true);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `topicName` to `{topic_id: topicName}` in the call to `updateFollowTopicDiscoveryContext` seems incorrect. If `topic_id` is expected to be an ID, passing the `topicName` might lead to incorrect behavior. Please verify this.

```diff
-     updateFollowTopicDiscoveryContext(false, {topic_id: topicName}, true);
+     updateFollowTopicDiscoveryContext(false, {topic_id: topicId}, true);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->